### PR TITLE
ReadFileJob: Removing "Loading" from message

### PR DESCRIPTION
**Pull request by thopiekar**
We already have "Loading" in the message's title.
In my opinion one "Loading" too much..

### CODE REVIEW

File: UM/FileHandler/ReadFileJob.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def run(self):
                 result_message.show()
             return
 
-        self._loading_message = Message(i18n_catalog.i18nc("@info:status Don't translate the XML tag <filename>!", "Loading <filename>{0}</filename>", self._filename),
+        self._loading_message = Message(i18n_catalog.i18nc("@info:status Don't translate the XML tag <filename>!", "<filename>{0}</filename>", self._filename),
```
Comment (fieldOfView) [inline UM/FileHandler/ReadFileJob.py]: Then why run it through i18n at all?

Comment (thopiekar) [reply UM/FileHandler/ReadFileJob.py]: 🤦‍♂️

**Comment by thopiekar** [comment]
@fieldOfView Thank you for your first review!

**Comment by Ghostkeeper** [comment]
image
Looks nice.

**Comment by thopiekar** [comment]
Hartelijk bedankt

**Pull request merged**